examples/alerts/tests/availability_test.yaml

```diff
@@ -0,0 +1,45 @@
+rule_files:
+  - /prometheus/availability-rules.yaml
+
+evaluation_interval: 1m
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: istio_requests_total{job="ingress-metrics-proxy",response_code="500"}
+        values: "0+0x30 0+10x30"
```

Comment:
i've always found this syntax to be complex to decode.
I wonder if we can add comments to explain in simpler terms what's going on.
Something like:

```yaml
# 0+0x30 => 0, 0, 0, ...0 (total 31 entries)
# then,
# 0+10x30 => 0, 10, 20, ...300 (total 31 entries) 
```

for below, might look like:

```yaml
# 0+1x30 => 0, 1, 2, ...30 (total 31 entries)
# then
# 31+100x30 => 31, 131, 231,...3031 (total 31 entries)
```